### ISSUE TITLE
Switch directions options language to locale

### ISSUE THREAD

**Issue by dgearhart** [valhalla/odin#247]

**Comment by rossanag** [comment]
I would appreciate this options, is there any help I could give to translate it to spanish? Best!

**Comment by dgearhart** [comment]
@rossanag I was hoping you could provide a Spanish narrative language file.
If you are able, here are the instructions:
https://github.com/valhalla/odin/blob/master/locales/README.md

The file name for the Spanish/Spain narrative language should be:
es-ES.json
 
The posix_locale value in the es-ES.json file should be:
es_ES.utf8

Please let me know if you are not able or if you have any questions.
Thank you

**Comment by rossanag** [comment]
Yes, I am willing to do it. I can put my hands on it next week, so very likely I will have questions to ask to you. I contact you next week. 
Best!

**Comment by dgearhart** [comment]
Great - thank you

**Comment by rossanag** [comment]
I began to translate the instructions to the spanish language. I will comment any issue or when I finish it. I guess it will be tomorrow or on Friday.
Best!

**Comment by dgearhart** [comment]
Excellent - thank you

**Comment by rossanag** [comment]
Just in case I post the same here:
https://github.com/valhalla/odin/issues/321

**Comment by darkwizzard** [comment]
I want collaborate with spanish translation, where i can find the advance?

**Comment by rossanag** [comment]
I finished the spanish translation. I had some problems to run the tests. I will do the Pull Request as soon as possible. Best!

**Comment by kevinkreiser** [comment]
Excellent, feel free to make a PR whenever you have the time. I'd be happy to help you figure out whats going on!

**Comment by darkwizzard** [comment]
@rossanag, can you show the translation for test? Can we collaborate with it?..

**Comment by dgearhart** [comment]
@rossanag any chance you could PR soon?
@darkwizzard has offered to collaborate
we can help with test issues

**Comment by rossanag** [comment]
Yes, I am with the hands on it. The tanslation is alredy done. I couldn't run the test. I will write later.

**Comment by rossanag** [comment]
Just in case I attach the json file
[es-ES.json.zip](https://github.com/valhalla/odin/files/412095/es-ES.json.zip)

**Comment by dgearhart** [comment]
@rossanag thank you
I submitted PR to your fork that fixes the test issues

**Comment by rossanag** [comment]
Thank you!

**Comment by darkwizzard** [comment]
How can i use spanish language in on the road (Android)? thanks.

**Comment by gknisely** [comment]
@ecgreb can you answer @darkwizzard 's question?

**Comment by ecgreb** [comment]
Setting the language is not currently supported out of the box in OTR Android but could be added by extending and overriding a couple of classes and methods that are used to build the request.

https://github.com/mapzen/on-the-road_android/issues/89

Official support for setting the language has been added to the roadmap for the next release.
